home.html

```diff
@@ -0,0 +1,24 @@
+<div class="w3-container">
```

Comment:
Since this page only lists all contacts, it's name should be something like "contacts-list.html" for example.